intezer_sdk/util.py

```diff
@@ -0,0 +1,119 @@
+import collections
+import logging
+from typing import List, Tuple, Optional
+
+from intezer_sdk.analysis import Analysis
+
+logger = logging.getLogger(__name__)
+
+
+def get_analysis_summary(analysis: Analysis, options: dict) -> str:
+    result = analysis.result()
+    metadata = analysis.get_root_analysis().metadata
+    verdict = result['verdict'].lower()
+    sub_verdict = result['sub_verdict'].lower()
+    ignore_emojis = options.get('ignore_emojis', False)
+
+    note = (f'Intezer Analyze File Scan\n'
+            f'=========================\n\n')
+
+    if verdict == 'malicious':
+        emoji = get_emoji(verdict, ignore_emojis)
+        main_family, gene_count = get_analysis_family(analysis, ['malware', 'malicious_packer'])
+    elif verdict == 'trusted':
+        emoji = get_emoji(verdict, ignore_emojis)
```

Comment:
you can do it once after the if-else. You can also remove the flag from the function itself

Reply:
done